frontend/containers/project-form-pages/pages/general-information.tsx

```diff
@@ -25,11 +26,14 @@ const GeneralInformation = ({
   errors,
   control,
   getValues,
+  resetField,
 }: ProjectFormPagesProps<ProjectForm>) => {
   const [showInvolvedProjectDevelopers, setShowInvolvedProjectDevelopers] = useState(false);
+  const [locationsFilter, setLocationsFilter] =
+    useState<{ country?: string; department?: string }>();
```

Comment:
You could avoid possible mistakes by doing this:
```suggestion
    useState<{ country: string; department: string }>({ country: undefined, department: undefined });
```